consts.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Actually, it looks like the only problem is that this file was added by accident (`src/consts.js` is the right one).